### PR TITLE
fix invalid metadata name in helm chart files (#1294)

### PR DESCRIPTION
**Pull request by cwen0**
### What problem does this PR solve?

cherry-pick #1294

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: helm/chaos-mesh/templates/controller-manager-rbac.yaml [hunk 1]
```diff
@@ -14,9 +14,9 @@ metadata:
 ---
 # roles
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (YangKeao) [inline helm/chaos-mesh/templates/controller-manager-rbac.yaml:17]: Why does this PR modify this line?

If you need, please cherry-pick #994

Comment (cwen0) [reply helm/chaos-mesh/templates/controller-manager-rbac.yaml:17]: Okay, I have cerated this pr to cherry-pick it https://github.com/chaos-mesh/chaos-mesh/pull/1296

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**